prover/t/unit/match-assoc.k

```diff
@@ -120,4 +121,19 @@ module TEST-MATCH-ASSOC
                           )
              )
        .Declarations
+  // Match multiple occurances of a variable
+  rule test("match-assoc", 10)
+    => symbol c ( Data ) : Data
```

Comment:
How does this work? I thought that `#matchAssoc`, which is a function, is evaluated before the symbol declaration.